### PR TITLE
Prov legislature pages

### PR DESCRIPTION
**Pull request by xybrnet**
An initial stab at the provincial legislature pages. We need to confirm which pages we'll be linking to on the provincial websites.

prov_leg_listing
prov-leg-detail

### CODE REVIEW

**Comment by longhotsummer** [comment]
Looks good, @xybrnet. Will follow-up in slack.

**Comment by xybrnet** [comment]
@longhotsummer 
This pull request sees to the PL feature requests, is ready for review.

File: pmg/templates/provincial/detail.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
+{% extends "base.html" %}
+
+{% block title %}{{ province.name }} Parliament | PMG{% endblock %}
```
Comment (longhotsummer) [inline pmg/templates/provincial/detail.html]: Parliament -> Legislature?

File: pmg/templates/provincial/western_cape.html [hunk 2]
```diff
@@ -1,14 +1,35 @@
 {% extends "base.html" %}
 
-{% block title %}{{ province }} Parliament | PMG{% endblock %}
+{% block title %}{{ province.name }} Parliament | PMG{% endblock %}
```
Comment (longhotsummer) [inline pmg/templates/provincial/western_cape.html]: Parliament -> Legislature?

File: pmg/utils.py [hunk 3]
```diff
@@ -71,3 +71,57 @@ def externalise_url(url):
 
     scheme = 'http' if app.config['DEBUG'] else 'https'
     return '%s://%s/%s' % (scheme, request.host, url)
+
+
+def get_provincial_legislatures():
```
Comment (longhotsummer) [inline pmg/utils.py]: Shouldn't this now be pulled from the DB?

**Comment by longhotsummer** [comment]
Looks good, Damian. Mostly minor comments.

The big one is whether we can replace get_provincial_legislatures with a call to the db? Not even going through the API, just `House....`

**Comment by xybrnet** [comment]
@longhotsummer 

Thanks Greg. I've made the changes to replace `get_provincial_legislatures` and use Houses in the DB. Please let me know what you think.

**Comment by longhotsummer** [comment]
Looks good!

**Pull request merged**